### PR TITLE
Bom bom bom

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Service/Modules/ManufacturingTimingsModule.cs [hunk 1]
```diff
@@ -14,26 +14,25 @@
...
             this.manufacturingTimingsService = manufacturingTimingsService;
-            this.authorisationService = authorisationService;
 
             this.Get("/production/reports/manufacturing-timings", _ => this.GetTimingsReport());
             this.Get("/production/reports/manufacturing-timings/export", _ => this.GetTimingsExport());
         }
 
         private object GetTimingsReport()
         {
+            var x = this.manufacturingTimingsService.GetTimingsForAssembliesOnABom("SK HUB");
```
Comment (richardiphillips) [inline src/Service/Modules/ManufacturingTimingsModule.cs:34]: is this line needed?

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 2]
```diff
@@ -157,7 +157,11 @@ public class ServiceDbContext : DbContext
         public DbQuery<OsrRunMaster> OsrRunMaster { get; set; }
 
         public DbQuery<PtlMaster> PtlMaster { get; set; }
-        
+
+        public DbSet<Bom> Boms { get; set; }
+
+        public DbSet<BomDetail> BomDetails { get; set; }
```
Comment (richardiphillips) [inline src/Persistence.LinnApps/ServiceDbContext.cs:163]: do you use the BomDetails DbSet and repository?

**Review by richardiphillips — APPROVED** [review]
funky stack. like it

File: src/Persistence.LinnApps/ServiceDbContext.cs (hunk 2 above)
Comment (lewisrenfrew) [reply src/Persistence.LinnApps/ServiceDbContext.cs:163]: nope

File: src/Service/Modules/ManufacturingTimingsModule.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Service/Modules/ManufacturingTimingsModule.cs:34]: No, was just using that as a quick test. I think I'll do a csv endpoint for this

**Pull request closed without merging**